Testing/WinMLRunnerTest/WinMLRunnerTest.cpp

```diff
@@ -6,6 +6,7 @@
 #include <Winbase.h>
```

Comment:
Would be great to have PreprocessImage tests for DenseNet121 as well! The onnx zoo model is ~32MB large so it's under Github's file limit of 100MB

Reply:
I can add this if you like... or do you want to use a specific version of DenseNet?  I have opset 7 and opset 8 in fp32 and fp16 readily available (from onnx model zoo).

Reply:
I think for testing that image preprocessing works, I think using DenseNet ai.onnx version 7 in fp32 would be sufficient. Do you agree?

Reply:
I think that will be fine.  I will go ahead and add a model and test for this.

Reply:
Thanks!